src/components/SettingDialog.vue

```diff
@@ -49,25 +49,128 @@
                 />
               </q-card-actions>
             </q-card>
-
-            <!-- File Encoding Card -->
+            <!-- Saving Card -->
             <q-card class="setting-card">
               <q-card-section class="bg-blue">
-                <div class="text-h5">文字コード</div>
+                <div class="text-h5">保存</div>
               </q-card-section>
+              <q-list>
+                <q-expansion-item
+                  dense
+                  dense-toggle
+                  expand-separator
+                  group="saving-group"
+                  label="文字コード"
+                  expand-icon-class="text-black"
+                >
+                  <q-card>
+                    <q-card-section>
+                      <q-btn-toggle
+                        :model-value="savingSetting.fileEncoding"
+                        toggle-color="blue"
+                        :options="[
+                          { label: 'UTF-8', value: 'UTF-8' },
+                          { label: 'Shift_JIS', value: 'Shift_JIS' },
+                        ]"
+                        @update:model-value="
+                          handleSavingSettingChange('fileEncoding', $event)
+                        "
+                      />
+                    </q-card-section>
+                  </q-card>
+                </q-expansion-item>
 
-              <q-separator />
+                <q-expansion-item
+                  dense
+                  dense-toggle
+                  expand-separator
+                  group="saving-group"
+                  label="書き出し先を固定"
+                  expand-icon-class="text-black"
+                >
+                  <q-card>
+                    <q-card-section>
+                      <q-toggle
+                        name="enabled"
+                        align="left"
+                        dense
+                        color="blue"
+                        :model-value="savingSetting.fixedExportEnabled"
+                        :label="
+                          savingSetting.fixedExportEnabled ? '有効' : '無効'
```

Comment:
これをクリックすると有効になるのか、今有効なのか、ユーザーが混乱しそうです。
有効・無効どちらの場合も「固定する」というラベルにするのはどうでしょう。

It seems to confuse users whether clicking on this will enable it, or whether it is enabled now.
How about labeling it as "fix" for both enabled and disabled?